tests/test_refresh_source_authority.py

```
@@ -0,0 +1,231 @@
+
+# ---------------------------------------------------------------------------
+# Step orchestration
+# ---------------------------------------------------------------------------
+
+
+class TestStepOrchestration:
+    def test_all_three_steps_invoked_with_pass_through_args(self, tmp_path):
+        vault = _make_vault(tmp_path)
+        captured: dict[str, list[str]] = {}
+
+        def fake(name):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `fake`

(ANN202)